Sources/AppcuesKit/Presentation/Actions/Appcues/AppcuesRequestPushAction.swift

```diff
@@ -33,13 +33,12 @@ internal class AppcuesRequestPushAction: AppcuesExperienceAction {
 
             let pushMonitor = appcues.container.resolve(PushMonitoring.self)
             let analyticsPublisher = appcues.container.resolve(AnalyticsPublishing.self)
+            analyticsPublisher.publish(TrackingUpdate(
+                type: .event(name: Events.Device.deviceUpdated.rawValue, interactive: false),
+                isInternal: true
+            ))
```

Comment:
sorry im having trouble following the logic, assuming it works but does this trigger an update on the result handler, and then L41 would trigger another since the current status changed?

Reply:
This shouldn't be here at all, I must've inadvertently pasted it. Should just be this, which hopefully makes more sense. Thanks for catching this!

```swift
let pushMonitor = appcues.container.resolve(PushMonitoring.self)
pushMonitor.refreshPushStatus(publishChange: true) { _ in
    completion()
}
```